devices.js

```diff
@@ -156,8 +178,16 @@ const preset = {
             ...preset.light_onoff_brightness_colortemp_colorxy,
             toZigbee: preset.light_onoff_brightness_colortemp_colorxy.toZigbee.concat([tz.ledvance_commands]),
         },
+        light_onoff_brightness_colorxyhs: {
```

Comment:
Doesn't seem to be used, why add it?

Reply:
I added the preset for all types to make it easier to actually add the function later. After what @sjorge wrote about Osram/Ledvance bulbs i added the functionality to those too, so now they are actually being used, too.

Reply:
It could be an option to remove the .._colorxy if
- all ledvance bulbs support xy and hs
- we do not expect to have a bulb in the future which only accepts either hs or xy (rgb)

This holds true for hue and general color too.

A.

Reply:
> we do not expect to have a bulb in the future which only accepts either hs or xy (rgb)

Please remove any unused configurations. It's easy to re-add in the future.

Reply:
Done. Since "colorxyhs" seems to become the new default, i also refactored so that the default expose and presets are now
..._color, rather than ..._colorxyhs
At the same time, i have removed the preset.light_onoff_brightness_colortemp_colorhs - it was no longer in use. 
I did not touch the preset.light_onoff_brightness_colortemp_colorxy - too many devices use that which i have no information on.